### ISSUE TITLE
Leaflet doesn't sanitize field names

### ISSUE THREAD

**Issue by tomchadwin** [tomchadwin/qgis2web#316]
It seems that OL3 does, so there should be a function for us to call...

**Comment by tomchadwin** [comment]
Not as simple as that. OL3 manages this because it uses `feature.get()` with a quoted argument, so it handles special characters:

'''javascript
feature.get("% change")
```

Leaflet uses dot notation, so it fails:

'''javascript
feature.properties.% change
```

Will have to investigate further.

**Comment by tomchadwin** [comment]
All is well - Leaflet supports `['']` notation. Fixed in master. Please test.

**Comment by tomchadwin** [comment]
Released as 1.13.1 and pushed to QGIS plugins repo.
